modules/va_facilities/spec/requests/nearby_request_spec.rb

```diff
@@ -175,6 +175,20 @@ def query_params(url)
         expect(health_service['wait_times']['effective_date']).to eq('2018-03-05')
       end
     end
+
+    it 'only queries health facilities ignoring type paramater' do
```

Comment:
Do we specify to supply a type parameter of health?

Reply:
I removed the reference to the `type` parameter from the documentation. This test was just to verify that the response would be unaffected if a consumer supplied the parameter anyway.